blazor/getting-started/blazor-web-app.md

```diff
@@ -15,8 +15,6 @@ This article provides a step-by-step instructions for building `Blazor Web App`
 
 [Blazor Playground](https://blazor.syncfusion.com/documentation/blazor-playground/overview) allows you to interact with our Blazor components directly in your web browser without need to install any required NuGet packages. By default, the `Syncfusion.Blazor` package is included in this.
 

```

Comment:
Due to CI failure, we removed this line. 
![image](https://github.com/syncfusion-content/blazor-docs/assets/33709971/9641f103-b264-49dd-bc93-2214c8ea74bd)